ballerina/to_json_schema.bal

```diff
@@ -45,8 +45,12 @@ isolated function generateJsonSchemaForTypedesc(typedesc<json> td, boolean nilab
 
     if isArray {
         typedesc<json> arrayMemberType = getArrayMemberType(<typedesc<json[]>>td);
+        map<json>? ann = arrayMemberType.@Schema;
```

Comment:
Why this need to change? https://github.com/ballerina-platform/module-ballerinax-np/pull/11#discussion_r1985157246

Reply:
We need it there also, for example, record-typed field in a record.

Reply:
Updated

Reply:
Can't see the change?

Reply:
https://github.com/ballerina-platform/module-ballerinax-np/blob/19e7202230cf0fa7b9df6210e512fb517a7a2af6/ballerina/to_json_schema.bal#L106
Here I have checked whether annotation avaiable or not